src/extensions/extensionsHelper.ts

```diff
@@ -369,6 +380,26 @@ export async function promptForValidInstanceId(instanceId: string): Promise<stri
   return newInstanceId;
 }
 
+/**
+ * Prompts for a valid repo URI.
+ */
+export async function promptForValidRepoURI(): Promise<string> {
+  let repoIsValid = false;
+  let extensionRoot = "";
+  while (!repoIsValid) {
+    extensionRoot = await promptOnce({
+      type: "input",
+      message: "Enter the repo URI where this Extension's source code is located:",
+    });
+    if (!repoRegex.test(extensionRoot)) {
+      logger.info("Repo URI follows this format: https://github.com/<user>/<repo>");
```

Comment:
```suggestion
      logger.info("Repo URI must follow this format: https://github.com/<user>/<repo>");
```

Reply:
I had exactly that originally but Will thought the wording was too aggressive as an error so I changed it to "follows".